src/stub.js

```diff
@@ -1,6 +1,6 @@
 export const UserStub = {
   id: 1,
-  username: "Dick",
+  username: "User Name",
   first_name: "",
   last_name: "",
   email: "r.moore@theta-llc.com",
```

Comment:
Probably should also give a fake email as well, but not necessarily a blocking change.